packages/esm-patient-search-app/src/patient-search-page/refine-search/person-attribute-field.component.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { ComboBox, InlineLoading, InlineNotification, TextInput, TextInputSkeleton } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { useAttributeConceptAnswers, useLocations, usePersonAttributeType } from './person-attributes.resource';
+import { type AdvancedPatientSearchState, type SearchFieldConfig } from '../../types';
+import styles from './search-field.scss';
+
+export interface PersonAttributeFieldProps {
+  field: SearchFieldConfig;
+  formState: AdvancedPatientSearchState;
+  inTabletOrOverlay: boolean;
+  isTablet: boolean;
+  onInputChange: (fieldName: string) => (evt: { target: { value: string } } | { name: string }) => void;
+}
+
+export function PersonAttributeField({ field, formState, isTablet, onInputChange }: PersonAttributeFieldProps) {
+  const { t } = useTranslation();
+  const { data: personAttributeType, isLoading, error } = usePersonAttributeType(field.attributeTypeUuid);
+
+  const formatField = useMemo(() => {
+    if (!personAttributeType || isLoading) {
+      return <TextInputSkeleton />;
+    }
```

Comment:
If loading, then we should show a skeleton, but in case of an error, we need to show an inline notification rather than showing skeleton.